src/app/dataset-block/metadata-block/components/block-navigation/block-navigation.component.html

```diff
@@ -1 +1,72 @@
-<p>block-navigation works!</p>
+<div class="text-center mb-4 position-relative align-items-center">
+    <input
+        class="form-control placeholder-padding-left"
+        type="text"
+        name="search-hash"
+        [(ngModel)]="searchHash"
+        autocomplete="off"
+        placeholder="Search by hash"
+    />
+    <svg-icon name="search"></svg-icon>
```

Comment:
Please add some kind of icon to the right of the search input to clear the filter

Reply:
Added icon to clear filter